artemis-core-client/src/main/java/org/apache/activemq/artemis/core/protocol/core/impl/wireformat/SessionBindingQueryResponseMessage_V2.java

```diff
@@ -25,16 +25,21 @@
 
    protected boolean autoCreateQueues;
 
+   protected boolean autoCreateQueuesDurable;
```

Comment:
Why you are adding this if this is not being written anywhere?

Also... this change is wrong.. you would need a _V5 with the new field being written...

But honestly... there has been so any versions on SessionBindingQueryResponse... can you add a V5 with a hashMap for new fields? in the future we would just add more metadata instead of changing the packet.

Reply:
I personally don't see why this is needed at all, if this is meant to be a server side change only.